packages/platform-ios/template/ios/bundle-react-native-code-and-images.sh

```diff
@@ -0,0 +1,16 @@
+set -e
+
+if [[ -f "$PODS_ROOT/../.xcode.env" ]]; then
+    source "$PODS_ROOT/../.xcode.env"
+fi
+
+if [[ -f "$PODS_ROOT/../.xcode.env.local" ]]; then
+    source "$PODS_ROOT/../.xcode.env.local"
+fi
+
+export CONFIG_CMD="dummy-workaround-value"
+export CLI_PATH="$("$NODE_BINARY" --print "require('path').dirname(require.resolve('@rnef/cli/package.json')) + '/dist/src/bin.js'")"
+
+WITH_ENVIRONMENT="$REACT_NATIVE_PATH/scripts/xcode/with-environment.sh"
+REACT_NATIVE_XCODE="$REACT_NATIVE_PATH/scripts/react-native-xcode.sh"
+/bin/sh -c "$WITH_ENVIRONMENT $REACT_NATIVE_XCODE"
```

Comment:
i'm curious why it's like this

the previous lines are using stuff like `NODE_BINARY` which is set in `WITH_ENVIRONMENT`, so it'd be better to set it before calling this script.

the `source .xcode.env.local` etc. already seem to be done inside the `WITH_ENVIRONMENT` script.


```diff
- shellScript = "bash -l -c \"${PROJECT_DIR}/bundle-react-native-code-and-images.sh\"\n";
+ shellScript = "bash -l -c \"$REACT_NATIVE_PATH/scripts/xcode/with-environment.sh\" \"${PROJECT_DIR}/bundle-react-native-code-and-images.sh\"\n";
```

and then here:

```suggestion
set -e

export CONFIG_CMD="dummy-workaround-value"
export CLI_PATH="$("$NODE_BINARY" --print "require('path').dirname(require.resolve('@rnef/cli/package.json')) + '/dist/src/bin.js'")"

"$REACT_NATIVE_PATH/scripts/react-native-xcode.sh"
```

Also would be good to add a comment on top of the `CONFIG_CMD`.

Reply:
I'd also suggest not naming the script `bundle-react-native-code-and-images.sh` since one of the points to extract it to a separate script is so that it's easier to do more things, e.g., upload sourcemap after bundling.

Reply:
Struggling to come up with a good one though 😅

Reply:
I guess the challenge comes with having a name that suggests what you can put inside in case you need to.

Reply:
@szymonrybczak could you please share what we're working around here?

Reply:
@adhorodyski we're working around flawes `--config-cmd` handling in `react-native-xcode.sh`. 
I'd go with `react-native-xcode.sh` as a name and put it in a template. this will require to adjust migration steps to copy that file as well

Reply:
Should we leave `00DD1BFF1BD5951E006B06BC /* Bundle React Native code and images */ = {` as is or rename the step as well?

Reply:
Let's keep this name to avoid migration scope